Files/Strings/en-US/Resources.resw

```diff
@@ -508,7 +508,7 @@
     <value>Refresh (F5)</value>
   </data>
   <data name="NavSearchButton.ToolTipService.ToolTip" xml:space="preserve">
-    <value>Search</value>
+    <value>Search (Ctrl+F)</value>
```

Comment:
```suggestion
    <value>Search (Ctrl + F)</value>
```
Other shortcuts have a space before and after the `+` character. Perhaps we can change that to be consistent with other apps but that can be done in a different pull request.